2.0/.sti/bin/usage

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
+
+cat <<EOF
+This is a STI ruby-2.0 ${DISTRO} base image:
+To use it, install STI: https://github.com/openshift/source-to-image
+
+Sample invocation:
+
+sti build git://github.com/jhadvig/sinatra-app-example openshift/ruby-20-${DISTRO}7 sinatra-app
```

Comment:
use the contextDir and point to the test app in this repo instead of your personal repo.